stdlib/http/src/main/ballerina/src/http/websocket/websocket_client.bal

```diff
@@ -46,12 +44,13 @@ public type WebSocketClient client object {
         return externWSInitEndpoint(self);
     }
 
-    # Push text to the connection.
+    # Pushes text to the connection.
     #
-    # + data - Data to be sent, if byte[] it is converted to a UTF-8 string for sending
+    # + data - Data to be sent. If it is a byte[], it is converted to a UTF-8 string for sending
     # + finalFrame - Set to `true` if this is a final frame of a (long) message
-    # + return  - `error` if an error occurs when sending
-    public remote function pushText(string|json|xml|boolean|int|float|byte|byte[] data, 
+    # + return  - Returns an `error` if an error occurs while sending the text message to the server and that message
```

Comment:
Do we need `Returns` here?
Shall we make it `an `error` if an error occurs while sending the text message ....`

Reply:
Check all places